### PR TITLE
Reformatted to PEP8

### PR DESCRIPTION
**Pull request by harleygn**
Sorted those dodgy variable and function names

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -1,294 +1,318 @@
...
-          PlayerTiles, PlayerScore, PlayerTilesPlayed = UpdateAfterAllowedWord(Choice, PlayerTiles, PlayerScore, PlayerTilesPlayed, TileDictionary, AllowedWords)
-          NewTileChoice = GetNewTileChoice()
-      if not ValidWord:
+        player_one_tiles = "BTAHANDENONSARJ"
+        player_two_tiles = "CELZXIOTNESMUAA"
+    while player_one_tiles_played <= max_tiles_played and player_two_tiles_played <= max_tiles_played and len(
+            player_one_tiles) < max_hand_size and len(player_two_tiles) < max_hand_size:
+        player_one_tiles, player_one_tiles_played, player_one_score, tile_queue = have_turn("Player One", player_one_tiles, player_one_tiles_played, player_one_score, tile_dictionary, tile_queue, allowed_words, max_hand_size, no_of_end_of_turn_tiles)
         print()
-        print("Not a valid attempt, you lose your turn.")
```
Comment (max-kov) [inline main.py:215]: Why was this changed?

Comment (harleygn) [reply main.py:215]: Not sure, but looks to be identical

**Comment by harleygn** [comment]
This is the original code, without any changes except for the reformatting. As a result, it is heavily out of date but still has potential use.
